### ISSUE TITLE
[Suggestion] Improve In-game Organization for Blueprints

### ISSUE THREAD

**Issue by Kadzar** [tsunamayo/Starship-EVO#3198]
### Short Description of the issue:

One thing I've noticed after playing the game for a while, having made several blueprints and subscribed to several builds on the workshop is how the blueprint submenu can get crowded if you have a lot of them, and it even seems to load in slower.

I don't know if you or anyone else has a better idea of how to solve this, but maybe some kind of in-game folder system would help. Then we could sort things how we want. I have no idea how feasibly or non-feasible this is, but this was the solution that immediately came to my mind.

I'd just like something to cut down on all the blueprint clutter, because, currently, the only way I can think of to deal with it is to put blueprints I'm not using at the moment in some other folder (which also isn't great for users who aren't familiar with the file locations).

**Comment by tsunamayo** [comment]
Yes it is definitely in the to-do list, that is just time consuming and time not spend on somewhere else (hum survival hum).

**Comment by Kadzar** [comment]
Understandable. It's just good to know such a thing will happen eventually.
